lib/OpenQA/Script/CloneJobSUSE.pm

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018-2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package OpenQA::Script::CloneJobSUSE;
+
+use Mojo::Base -strict, -signatures;
+
+use Data::Dump 'pp';
+use Exporter 'import';
+use LWP::UserAgent;
+
+
+our @EXPORT = qw(
+  collect_incident_repos
+  detect_maintenance_update
+);
+
+sub collect_incident_repos ($settings) {
```

Comment:
This can work correctly now only for aggregate jobs right ?

Reply:
+ be aware of https://progress.opensuse.org/issues/111710

Reply:
As I understand, I can look up MAINT_TEST_REPO and INCIDENT_REPO only after this [111710](https://progress.opensuse.org/issues/111710) implementation. Instead of killing job I have modified the code to print the list of maintenance updates that have been released.